libs/npm/core/src/components/form/wysiwyg/styled/Wysiwyg.tsx

```diff
@@ -3,6 +3,27 @@ import styled from 'styled-components';
 import { IWysiwygProps } from '../Wysiwyg';
 
 export const Wysiwyg = styled.div<IWysiwygProps>`
+  .exit {
+    float: right;
+    &:hover {
+      cursor: pointer;
+      transform: scale(1.1);
+    }
+  }
+  dialog {
+    border: none;
+    width: 100%;
+    height: 85%;
+    .frm-in {
+      height: 85%;
+      .expand-button {
+        display: none;
```

Comment:
do not display the expand option when we have already expanded (otherwise user could open infinite instances and then have to close x instances)